docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
+        Example: `$ kubectl get pod example-pod1 example-pod2`
+        * To specify multiple resource types individually: `TYPE1/name1 TYPE1/name2 TYPE2/name3 TYPE<#>/name<#>`<br/>
+        Example: `$ kubectl get pod/example-pod1 replicationcontroller/example-rc1`
+   * To specify resources with one or more files: `-f file1 -f file2 -f file<#>`
+	 [Use YAML rather than JSON](config-best-practices.md) since YAML tends to be more user-friendly, especially for configuration files.<br/>
+     Example: `$ kubectl get pod -f ./pod.yaml`
+* `flags`: Specifies optional flags. For example, you can use the `-s` or `--server` flags to specify the address and port of the Kubernetes API server.<br/>
+**Important**: Flags that you specify from the command line override default values and any corresponding environment variables. *Tip*: Use short flags sparingly and only for the most frequently used options.<br/>
+For more information and examples about specifying flags, see the [kubectl conventions](../devel/kubectl-conventions.md#flag-conventions).
+
+If you need help, just run `kubectl help` from the terminal window.
+
+## Operations
+
+The following table includes short descriptions and the general syntax for all of the `kubectl` operations:
+
+Operation       | Syntax	|       Description
+-------------------- | -------------------- | --------------------
+`annotate`	| `kubectl annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]` | Add or update the annotations of a resource.
+`api-versions`	| `kubectl api-versions` | List the API versions in use.
+`apply`			| `kubectl apply` | Apply a configuration change to a resource by file or stdin.
+`attach`		| `kubectl attach POD -c CONTAINER` | Attach to a running container to send stdin from the terminal window.
+`cluster-info`	| `kubectl cluster-info` | Display endpoint information about the master and services in the cluster.
+`config`		| `kubectl config SUBCOMMAND` | Modifies kubeconfig files. See the sub-commands for details.
+`create`		| `kubectl create -f FILENAME` | Create a resource using a file or stdin.
+`delete`		| `kubectl delete ([-f FILENAME] | TYPE [(NAME | -l label | --all)])` | Delete resources either by filenames, label selectors, names, resource selectors, resources, or stdin.
+`describe`	| `kubectl describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)` | Display all the details for one or more resources.
+`edit`		| `kubectl edit (RESOURCE/NAME | -f FILENAME)` | Edit the definition of a resource on the server.
+`exec`		| `kubectl exec POD [-c CONTAINER] -- COMMAND [args...]` | Execute a command against a container in a pod.
+`expose`		| `kubectl expose (-f FILENAME | TYPE NAME) [--port=port] [--protocol=TCP|UDP] [--target-port=number-or-name] [--name=name] [----external-ip=external-ip-of-service] [--type=type]` | Duplicate and expose a replication controller, service, or pod as a new Kubernetes service.
```

Comment:
`kubectl expose (-f FILENAME | TYPE NAME | TYPE/NAME) [--port=port] [--protocol=TCP|UDP] [--target-port=number-or-name] [--name=name] [----external-ip=external-ip-of-service] [--type=type] [flags]`

Suggest to keep "Take a replication controller, service, or pod and expose it as a new Kubernetes service."